test/transformation-test.coffee

```diff
@@ -45,7 +45,11 @@ parseApiBlueprint = (source, type, cb) ->
 
     sourcemap = result?.sourcemap
 
-    err = adapter.transformError(source, err)
+    if err
```

Comment:
Protagonist doesn't though, the test would start failing if that was the case and then this + the next few lines could be removed.